data/de/vsn-hafas-mgate.json

```diff
@@ -31,11 +31,12 @@
       "v": "5030100",
       "os": "iOS 13.3"
     },
-    "ver": "1.24",
+    "ver": "1.27",
```

Comment:
FYI: Up to a value of `1.42`, the server does not complain about not supporting the respective version, so we could even pick a higher value here.